crates/builder/src/prebuild/assembly.rs

```diff
@@ -39,8 +41,8 @@ impl Assembly {
         for sass in &self.sass {
             log::info!("Processing sass assembly '{name}'");
             let css = sass.process(info)?;
-            let filename = sass.file.file_name().unwrap();
-            let hash = sass.out.write_file(css.as_bytes(), &site_dir, filename)?;
+            let filename = sass.file.file_name().unwrap().replace("scss", "css");
```

Comment:
Minor thing: better to move this to the Sass class (where it is already done once) and to use the `file_stem()` with the `css` appended.